data/tree/node setter (mutable)/ec47f31f-a2f2-4f53-ab3e-a5345e5bca41.json

```diff
@@ -10,18 +10,19 @@
       "1670d7c3-36ce-42d6-8411-b90866ca411a": {
         "type": "(string or number)[]",
         "name": "path",
-        "description": "Specifies the location of the node."
+        "description": "Receives the path segments where the `node` should  be written to"
       },
       "5436f8ba-84e3-48e2-bd39-9ccaa8aeea9a": {
         "type": "any",
         "name": "node",
-        "description": "The node to be stored in the tree."
+        "description": "Receives the node to be written"
       }
     },
     "outputs": {
       "eb1f3b40-bfd7-4343-acd9-2adcdd6d5a8c": {
-        "type": "null",
-        "name": "written"
+        "type": "any[] or {string:any}",
+        "name": "written",
```

Comment:
I find this port name quite unfortunate as we usually use this style of port name for events triggered by asynchronous operations.
Maybe rename it to "modified tree" or "mutated tree" or just simply "tree"?